datalad/utils.py

```diff
@@ -1715,15 +1803,15 @@ def dlabspath(path, norm=False):
     if not isabs(path):
         # if not absolute -- relative to pwd
         path = op.join(getpwd(), path)
-    return normpath(path) if norm else path
+    return normpath(path) if norm else str(path)
```

Comment:
this changes "functionality" since before it could have returned original `Path`.  I wonder if there would be no effect on any use...

Reply:
The original path would only have been returned if it was absolute and `norm` was false, which is a rather specific situation for the return type to suddenly change.  The only time you could rely on getting a `Path` back would be instances in which calling the function was pointless to begin with.